### PR TITLE
George/refactor

### PR DESCRIPTION
**Pull request by george43g**
- jsonl complete refactor
- rename functions to remove `beta`
- cleanups

### CODE REVIEW

File: apps/backend-ops/src/jsonlArchive.ts [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+import { latestAnonDbFilename, loadAdminServices } from "@blockframes/firebase-utils";
+import { cleanDeprecatedData } from './db-cleaning';
+import { cleanStorage } from './storage-cleaning';
+import { copyAnonDbFromCi, readJsonlFile, restoreStorageFromCi } from '@blockframes/firebase-utils';
+import { generateFixtures } from './generate-fixtures';
+import { firebase } from '@env';
+const { storageBucket } = firebase();
+
+
+export async function prepareForTestingOld() {
```
Comment (phpgeekfr) [inline apps/backend-ops/src/jsonlArchive.ts:16]: maybe add `@deprecated` comment on each ?

File: libs/firebase-utils/src/lib/firestore/jsonlArchive.ts [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+import admin from 'firebase-admin';
+import { isArray, isEqual, isPlainObject, sortBy } from 'lodash';
+import { runChunks } from "../firebase-utils";
+import { META_COLLECTION_NAME } from "@blockframes/utils/maintenance";
+import { Firestore } from "../types";
+import { Queue } from "../queue";
+import { CollectionReference, DocumentReference } from "../types";
+
+
+export async function clearOld(db: FirebaseFirestore.Firestore) {
```
Comment (phpgeekfr) [inline libs/firebase-utils/src/lib/firestore/jsonlArchive.ts:16]: maybe add `@deprecated` comment on each ?

**Review by phpgeekfr — APPROVED** [review]
LGTM.
What is command line to interact with old jsonl methods?

**Comment by george43g** [comment]
There is no command, They're just in archive. To run them, we would have to add the command back in, but the code is there.

**Pull request merged**